scripts/tcs_listener.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+listener="${TCF_HOME}/examples/common/python/connectors/direct/tcs_listener/tcs_listener.py"
+# Read Listener port from config file
+# listener_port=`grep listener_port ${TCF_HOME}/config/tcs_config.toml | awk {'print $3'}`
```

Comment:
Why is this line commented? Can we read port from toml file instead of hardcoding it as 8080

Reply:
Because the tcs service (configured in docker/k8s/tcs.yaml) needs to know the port exposed by the container beforehand.

If not hardcoded, every update on the `listener_port` in config/tcs_config.yaml may render the tcs deployment inaccessible to the tcs service.

In the latest commit, I have worked around this by adding an option to read port from env as below 

https://github.com/hyperledger-labs/trusted-compute-framework/blob/ea7f78a8afc209bcc0d3fc5f27f3999b4e8e1ade/scripts/tcs_listener.sh#L5-L13